miden-lib/src/transaction/mod.rs

```diff
@@ -91,13 +91,13 @@ impl TransactionKernel {
     ///
     /// The initial stack is defined:
     ///
-    /// > [BLOCK_HASH, acct_id, INITIAL_ACCOUNT_HASH, NULLIFIER_COMMITMENT]
+    /// > [BLOCK_HASH, acct_id, INITIAL_ACCOUNT_HASH, INPUT_NOTES_COMMITMENT]
     ///
     /// Where:
     /// - BLOCK_HASH, reference block for the transaction execution.
     /// - acct_id, the account that the transaction is being executed against.
     /// - INITIAL_ACCOUNT_HASH, account state prior to the transaction, ZERO for new accounts.
-    /// - NULLIFIER_COMMITMENT, sequential hash of all input note's nullifiers.
+    ////- INPUT_NOTES_COMMITMENT, see `transaction::api::get_input_notes_commitment`.
```

Comment:
minor nit: there is an extra `/` in the comment.

Reply:
thanks, fixed